panda_agent/server.py

```diff
@@ -58,7 +59,9 @@ def StopRecording(self, request: pb.StopRecordingRequest, context):
         )
 
 def serve():
-    panda = Panda(generic='x86_64')
+    panda = Panda(arch='x86_64', qcow='/panda/shared/system_image.qcow2', mem='1024',
```

Comment:
Did you ever get around to moving defining the PANDA object to startAgent/ start container, or was that out of scope of this particular pull request?

Reply:
This was not really in the scope of this feature. I thought that was something being included in the backend cleanup branch, but I could be wronged so we may want to officially assign someone to take a look